src/renderer/components/+workloads-statefulsets/statefulsets.tsx

```diff
@@ -77,16 +74,3 @@ export class StatefulSets extends React.Component<Props> {
     );
   }
 }
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency